linera-execution/src/runtime.rs

```diff
@@ -445,6 +528,19 @@ where
         self.remaining_fuel.load(Ordering::Acquire)
     }
 
+    fn runtime_counts(&self) -> RuntimeCounts {
+        let remaining_fuel = self.remaining_fuel.load(Ordering::Acquire);
+        let num_reads = self.num_reads.load(Ordering::Acquire);
```

Comment:
I forgot… why is there only a `num_reads` but no `num_writes`?

Reply:
It is because we have a single write at the end of the block when the `.save()` is called.